### PR TITLE
v1.2.0 -> v1.2.2

### PR DESCRIPTION
**Pull request by pbx0**

### CODE REVIEW

File: multi-node/aws/pkg/config/templates/cloud-config-controller [hunk 1]
```diff
@@ -92,11 +92,17 @@ write_files:
       #!/bin/bash -e
       /usr/bin/curl -H "Content-Type: application/json" -XPOST -d @"/srv/kubernetes/manifests/kube-system.json" "http://127.0.0.1:8080/api/v1/namespaces"
 
-      for manifest in {kube-dns,heapster}-rc.json;do
+      for manifest in {kube-dns}-rc.json;do
           /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
           -d @"/srv/kubernetes/manifests/$manifest" \
           "http://127.0.0.1:8080/api/v1/namespaces/kube-system/replicationcontrollers"
       done
+      for manifest in {heapster}-rc.json;do
```
Comment (aaronlevy) [inline multi-node/aws/pkg/config/templates/cloud-config-controller]: The -rc should be replaced with -dep (or something to denote deployment)

File: multi-node/aws/pkg/config/templates/cloud-config-controller [hunk 2]
```diff
@@ -475,46 +481,93 @@ write_files:
...
-            },
-            "name": "heapster-v1.0.2",
+              "k8s-app": "heapster", 
+              "kubernetes.io/cluster-service": "true", 
+              "version": "v1.0.2"
+            }, 
+            "name": "heapster-v1.0.2", 
             "namespace": "kube-system"
-          },
+          },
```
Comment (aaronlevy) [inline multi-node/aws/pkg/config/templates/cloud-config-controller]: Nit: remove the trailing whitespaces added to file

**Comment by aaronlevy** [comment]
Only two minor comments. Otherwise lgtm

**Comment by pbx0** [comment]
Fixed up comments. This commit is causing a cluster to not come up for me at the moment, so still looking into that before I can merge.

**Comment by pbx0** [comment]
Looks like the issue with conformance testing was due to a change in our testing infrastructure. I'm now seeing some test failures but I'm somewhat confident sure its not related to this PR. I'm merging this for now and will continue looking into recent failures.

**Pull request merged**